### PR TITLE
schedule: Scroll to detailed card on match card click

### LINKED ISSUE #115: Scroll to Match Card when user tries to click on the match in overall view
We saw from beach nationals that there are a lot of dead clicks coming from the schedule page where users are trying to click on the badge on the top but nothing happens. We can make the badge clickable and when clicked we can scroll down to the match card which user clicked.
Screenshot 2023-10-09 at 11 44 38 PM

### PR DESCRIPTION
**Pull request by AnishKacham**
- [X] Scroll to detailed card
- [x] Flash the card that's brought into view

Fixes #115

### CODE REVIEW

**Comment by AnishKacham** [comment]
- Flashing of the card would not be perceived in case there are a lot of matches and smooth scroll takes more than 1.5s.
Didn't find any sure shot ways to check for smooth scroll finish.
https://github.com/w3c/csswg-drafts/issues/3744#issuecomment-685683932

**Comment by Joe2k** [comment]
@AnishKacham can you attach some screenshots of what you added new in this PR?

**Comment by AnishKacham** [comment]
Sure,
https://github.com/india-ultimate/hub/assets/79566582/1c294101-01e5-48df-bfcd-0aa34e0928af

**Comment by Joe2k** [comment]
GG Looks nice.

We are using MatchCard in other places also. If iirc we are using it twice in TournamentManager as well. Can you check those places too and flashes and scrolling if possible in this same PR? In those 2 places only 1 place will need scrolling when clicked another is just for showing the card with color without clickability. So try to make the MatchCard dynamic for both usecases.

Nit: Any other color possible other than yellow/orange for the flash :P Be nice if it can be the primary blue only and text all turns to white for visibility or any other color

**Comment by AnishKacham** [comment]
> In those 2 places only 1 place will need scrolling when clicked.

Will check this.

> Be nice if it can be the primary blue only and text all turns to white for visibility or any other color

True, I got carried away by some other places where this flash is used, I'm unable to remember where I saw this being used but knew it was yellow for sure. Will try some other color yeah. 🚀

**Comment by Joe2k** [comment]
Totally your call.. Try out different stuff and pick which you like. For dark too I thought the same combination(with darker blue bg) will work. But yeah only when you try only you'll know i guess...

**Comment by AnishKacham** [comment]
Here you go
https://github.com/india-ultimate/hub/assets/79566582/33a45c00-8858-4831-bd7e-74cb85aa59ce

File: frontend/src/components/TournamentManager.js
Comment (Joe2k) [inline frontend/src/components/TournamentManager.js:1]: At Line 945

It shouldn't be a button. So be good if we have both the functionality of a button and a badge in MatchCard as I mentioned in my previous comment.

Comment (Joe2k) [reply frontend/src/components/TournamentManager.js:1]: Otherwise GG

**Review by Joe2k — APPROVED** [review]

**Comment by punchagan** [comment]
Thanks for this, @AnishKacham ! Lovely work! :tada:

**Pull request merged**